### PR TITLE
add semtech to app index

### PR DESCRIPTION
**Pull request by dudmuck**
please add semtech project SWSD006 to ncs app index

### CODE REVIEW

File: index/dudmuck.json
Comment (wbober) [inline index/dudmuck.json:1]: Should this file be named "semtech.json"?

**Review by wbober — APPROVED** [review]

File: index/dudmuck.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                }
+            ],
+            "tags": [
+                "sidewalk",
+                "lora-basics-modem",
+                "CSS",
+                "FSK"
+            ],
+            "compatibleNcs": [
+                "v2.6.1"
```
Comment (FilipZajdel) [inline index/dudmuck.json:34]: Each release specifies sdk version it's compatible with. Please have a look at the suggestion I've put in another comment.

File: index/dudmuck.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "watchers": 1,
+            "stars": 1,
+            "forks": 0,
+            "apps": "samples/*",
+            "releases": [
+                {
+                    "date": "2024-08-08T14:30:49Z",
+                    "name": "v2.6.1",
+                    "tag": "v2.6.1"
+                }
```
Comment (FilipZajdel) [inline index/dudmuck.json:25]:
```suggestion
                    "sdk": "v2.6.1"
                }
```

File: index/dudmuck.json
Comment (FilipZajdel) [reply index/dudmuck.json:1]: As @wbober mentioned, this file should be named "semtech.json".

**Review by FilipZajdel — CHANGES_REQUESTED** [review]

File: index/dudmuck.json (hunk 2 above)
Comment (gbartnik) [reply index/dudmuck.json:25]: @dudmuck  please incorporate this suggestion.

File: index/dudmuck.json (hunk 1 above)
Comment (gbartnik) [reply index/dudmuck.json]: Done and updated

Comment (FilipZajdel) [reply index/dudmuck.json]: `"comparibleNcs"` is unnecessary at this point. Would you mind removing it, please?

Comment (gbartnik) [reply index/dudmuck.json]: Done.

File: index/semtech.json
Comment (FilipZajdel) [inline index/semtech.json:1]: Hi, please:

* update filename to `Lora-net.json` - the filename is then used to fetch the information from the repository to build the index, such as github.com/{organization-name}/{app-name}
* modify the file as suggested below - the index file has to conform to this [schema](https://github.com/nrfconnect/ncs-app-index/blob/main/resources/schema.json)
```suggestion
{
    "name": "Semtech",
    "description": "LoRa transceiver",
    "apps": [
        {
            "description": "Software Development Kit featuring demonstrations, drivers and middleware enabling development of connectivity and geolocation applications",
            "name": "SWSD006",
            "title": "Sidewalk & LoRaWAN Mullti-stack SDK for LR11xx (SWSD006)",
            "kind": "sample",
            "license": "Apache License 2.0",
            "apps": "samples/*",
            "releases": [
                {
                    "date": "2024-08-08T14:30:49Z",
                    "name": "v2.6.1",
                    "tag": "v2.6.1",
                    "sdk": "v2.6.1"
                }
            ],
            "tags": [
                "sidewalk",
                "lora-basics-modem",
                "CSS",
                "FSK"
            ]
        }
    ]
}
```

**Review by FilipZajdel — CHANGES_REQUESTED** [review]

File: index/semtech.json
Comment (gbartnik) [reply index/semtech.json:1]: @dudmuck please incorporate this suggestion.

**Review by FilipZajdel — APPROVED** [review]

**Pull request merged**